NURESTConnection.js

```diff
@@ -117,7 +118,12 @@ ResponseCodeEnum.initEnum({
     },
     ServiceUnavailable: {
         get code() { return 503; },
-        get errors() { return {}; }
+        get errors() {
+            return {
+                title: "Service Unavailable",
+                description: "System is in maintenance mode. Please try after sometime"
```

Comment:
maintenance mode is specific to VSD. js-bambou is a generic connection management library. lets use a generic description, same as title "Service Unavailable".

Reply:
done